shims/hdp25/assemblies/hdp25-shim/src/assembly/assembly.xml

```diff
@@ -10,7 +10,7 @@
       <outputDirectory>/</outputDirectory>
     </fileSet>
     <fileSet>
-      <directory>${oss.license.directory}</directory>
+      <directory>${oss.license.subdirectory}</directory>
```

Comment:
I don't see a reason to change this. A directory path is a directory path. "Subdirectory" doesn't add clarity to the property name here.